from types import TracebackType


class StrategyException(BaseException):

    def wrong_numbers(self):
        pass